flottform/where-are-you-at/src/routes/+page.svelte

```diff
@@ -99,3 +125,35 @@
 		Error connecting to friend
 	{/if}
 </div>
+
+<style>
```

Comment:
Can we move this styles into tailwind? 🤔  I think it would be easier then to work with in the future, cause all styles are in one place..